### DISCUSSION TITLE
Multiple simultaneous video feed: conversion from AVI to mp4

### DISCUSSION THREAD

**Discussion by samarth-wustl** [bonsai-rx/bonsai#1243] [Q&A]
Hi 

So I am using Bonsai to collect two video streams at the same time.
Currently I am storing them in AVI but it occupies a lot of space. I want to store them in mp4 format.

Can someone help me with what I need to do in order to save the files as mp4?
I have attached my current structure.

![bonsai_flow](https://user-images.githubusercontent.com/110621754/217112306-f3d96fa4-c9bb-4bdb-b4d9-735b667866cf.PNG)

Best,
Samarth

**Comment by bruno-f-cruz** [comment]
Hey,
Discussion #919 might be what you are looking for? If not let us know!

**Reply by samarth-wustl** [reply]
So I added the ffmpeg node! I can see both the simultaneuous feeds, however, a "The system cannot find the file specified" error shows up at the newly added ffmpeg node.

**Reply by bruno-f-cruz** [reply]
Can you attach the workflow you are running?

**Reply by samarth-wustl** [reply]
![bonsai_flow_updated](https://user-images.githubusercontent.com/110621754/217360820-f4248839-df76-4e89-91ba-cdac7bbdc3d3.PNG)

This is updated flow with the videoWriter (ffmpeg)
[Bonsai_mp4_try.zip](https://github.com/bonsai-rx/bonsai/files/10679245/Bonsai_mp4_try.zip)

**Reply by bruno-f-cruz** [reply]
There is only a layout file inside the zip I believe.

**Reply by samarth-wustl** [reply]
What kind of file would you need?

**Reply by bruno-f-cruz** [reply]
The workflow file. Should be a file with .bonsai extension

**Reply by samarth-wustl** [reply]
[updated_Bonsai.zip](https://github.com/bonsai-rx/bonsai/files/10699926/updated_Bonsai.zip)

Let me know if this works! 
Thanks for the help!

**Reply by bruno-f-cruz** [reply]
I am a bit confused. Your VideoWriter (FFMPEG) has no properties defined. You must give it a valid output name.
